operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -48,9 +50,11 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
    
 2. Update the LDAP settings in the customizations.yaml file.
 
-   1. (Optional) Add the LDAP CA certificate in the certs.jks section of customizations.yaml.
+   1. (Optional) Add the LDAP CA certificate in the certs.jks.b64 section of customizations.yaml.
 
       Follow step 8 under the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure.
+
+      Return to this procedure before running the command to inject and encrypt certs.jks.b64 into the customizations.yaml file in step 8.
```

Comment:
There is still a reference to /mnt/pitdata/${CSM_RELEASE} in step 8 that will not apply in the case of remoteISO. Should probably mention that anywhere they see /mnt/pitdata/${CSM_RELEASE} they should use ${CSM_DISTDIR}.